.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
can you please remove that file and maybe add a .gitignore ?

Reply:
i really wanted to have this file to easier integrate this into eclips ide

Reply:
well thats user-specific, so when everybody starts to commit his local configuration files we have a cluttered repo. Actually I think I should add a general .gitignore file that excludes things like .project and .idea and the like.

Reply:
i will remove

Reply:
is removed